### ISSUE TITLE
replacing long sleeves with short sleeves may cause problems

### ISSUE THREAD

**Issue by ZhangLei999** [Zheng-Chong/CatVTON#89]
I retrained with 120000 upper-data. During testing, it was found that when the model wore long sleeves, changing to short sleeves would cause problems. Will automatically replace short sleeves with long sleeves. Have you encountered it before？My human-parsing has been trained and the effect is not a problem.
![2024-12-09 10-48-39屏幕截图](https://github.com/user-attachments/assets/78522b63-6813-4bcb-b37b-815c5f487176)

**Comment by ZhangLei999** [comment]
The effect of changing short sleeves to long sleeves is very good. How did the original image recognize the long sleeves after being masked? I can guarantee that the test set has never appeared in the training set

**Comment by Zheng-Chong** [comment]
Please provide **masked person image** to further identify the problem.

**Comment by ZhangLei999** [comment]
Te mask in the example is on another computer. I found another demo with the same problem.
![2024-12-10 09-31-12屏幕截图](https://github.com/user-attachments/assets/3a8c223f-ae66-428f-8345-8f1e2392e80f)

**Comment by ZhangLei999** [comment]
Image before vton
![2024-12-10 09-35-08屏幕截图](https://github.com/user-attachments/assets/1deb2300-4fd8-405b-9daf-7c7e9fdf44ea)

**Comment by Zheng-Chong** [comment]
From the masked person image, a shadow of the sleeve may disclose some long sleeve information, you can try to mask the long-sleeve cloth by yourself and try again.

**Comment by ZhangLei999** [comment]
Not for this reason, I have done it before. Change clothes with long sleeves to clothes with short sleeves can easily lead to problems. I have tested IDM-VTON with the author's model and also encountered this issue.

**Comment by Zheng-Chong** [comment]
![image](https://github.com/user-attachments/assets/bbd108a3-ff5a-496a-afa9-97ea44caa645)

**Comment by Zheng-Chong** [comment]
From a technical point of view, inaccurate mask is the main cause of this problem. Manually providing a more accurate mask can alleviate this problem.

**Comment by ZhangLei999** [comment]
Thanks. I will verify it in the next few days

**Comment by KevinLiu0716** [comment]
@ZhangLei999 
Hello,

I hope this message finds you well. I have some questions regarding training and was wondering if you could help. If it's convenient for you, please feel free to contact me at this email: kevin071635@gmail.com

Thank you for your time and assistance!

Best regards,
Kevin
